src/index.ts

```diff
@@ -392,6 +421,9 @@ const main = async () => {
     {
       dqlStatement: z.string(),
     },
+    {
+      openWorldHint: true, // while we are not strictly talking to the open world here, the response from DQL could contain anything, so we treat it as open world
```

Comment:
> // while we are not strictly talking to the open world here, the response from DQL could contain anything, so we treat it as open world

Ah yes, that's also my current question. What is open world? Where does our closed world end and the openness begin? I'll put this on my weekend research list.

Reply:
It's a tricky one. I don't know how exactly an agent uses this information.

This is what the MCP C# SDK by Microsoft has to say:
> If `true`, the tool may interact with an unpredictable or dynamic set of entities (like web search). If `false`, the tool's domain of interaction is closed and well-defined (like memory access).
> The default is `true`.

So I'd say, `true` is probably correct here. (I see it similar to a web search.)